docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 Key Concepts
 ============
 
-The Product Delivery Model
------------------------------
-CumulusCI extends the concepts of the `Org Development Model <https://trailhead.salesforce.com/en/content/learn/modules/org-development-model>`_ and the `Package Development Model <https://trailhead.salesforce.com/en/content/learn/modules/sfdx_dev_model>`_ to form a new model, which we call the Product Delivery Model.
+Key concept. Developer jargon. 10-cent words that sound like a technical writer patting himself on the back. (Hi there!) We imagine your cursor subconsciously drifting its way to the hyperlink for the next chapter already. BUT in an effort to give you the tools to build and test customized features with CumulusCI, it's only right that we demystify the process by explaining the *how* and the *why* in order to help you achieve your goal of seamless automation.
```

Comment:
I see that you're trying to ease into this topic but this approach draws too much attention to the writer's voice. There's a time and a place for that; when content is dense, it's reassuring to hear the voice of someone knowledgeable right by your side. Seeing as this content is pretty high level and general, consider a more straightforward approach. "Let's review some important concepts when building and testing features using CumulusCI." Also be vigilant for creeping jargon like "seamless." =)

Reply:
In this case I am the perpetrator but my zeal for sesquipedalianism remains unbowed!

Reply:
Current thinking is that this entire section moves into the Introduction, right?

Reply:
@davidmreed Do you mean _all_ of Key Concepts moving into Introduction? As it stands, Intro is gonna be DENSE what with the explanation of CumulusCI and the Product Delivery Model.

Reply:
@lianeyee Fiiiiiiiiiine. Writer's voice quelled... for now.

For the record, I'm gonna blame @Br4nd0R for the phrase "seamless automation". (I personally prefer my automation to be rife with seams.)

Reply:
Entire paragraph sent out to pasture; adios, darlings. (Sir Arthur Quiller-Couch would be proud.)

Reply:
I mean the Product Delivery Model section moving.

Reply:
@davidmreed Yes, PDM will go into Intro. (Part of the reason why it's gonna be a bit of a beast to overhaul.)